cli/src/main/java/com/devonfw/tools/ide/tool/androidstudio/AndroidStudio.java

```diff
@@ -35,10 +33,8 @@ protected String getBinaryName() {
 
     if (this.context.getSystemInfo().isWindows()) {
       return STUDIO64_EXE;
-    } else if (this.context.getSystemInfo().isMac()) {
-      return STUDIO;
     } else {
-      return STUDIO_BASH_SCRIPT;
+      return STUDIO;
```

Comment:
This is a new feature of Intellij and older releases do not contain an `idea` script but only `idea.sh`.
So we should IMHO check if `idea.sh` exists but `idea` does not exist and make our choice dependent on that:
* `idea` script exists --> use `idea`
* `idea` script does not exist but `idea.sh` exists --> use `idea.sh`
* none of these scripts exists --> use `idea` or fail. I would suggest not to fail in case `getBinaryName()` is called for whatever reason before or without the tool being installed.

Same for android-studio.

Reply:
Thanks 👍